tests/unit/test_config.py

```diff
@@ -0,0 +1,3 @@
+project = 'VMyRPp1JmQtzw'
```

Comment:
This is configuration of one of my staging projects. This can be used for testing PR or can be replaced in future with some other project config.